### DISCUSSION TITLE
Exclude AppleTVs from iPhone remote list

### DISCUSSION THREAD

**Discussion by nicozagi** [maxileith/homebridge-appletv-enhanced#199] [General]
Hey there,

First off, thanks for both of your apps. Outstanding stuff! Finally my amplifier starts up with my AppleTV. 

I was wondering if there is a way to exclude the AppleTVs from Homebridge from the iPhone remote dropdown menu? Having each device twice can be confusing. Perhaps I also misunderstand the whole concept and I am supposed to kick the original AppleTVs from the Home app?

![IMG_3498](https://github.com/maxileith/homebridge-appletv-enhanced/assets/49094203/6f91ba61-1187-48ec-b918-139726bb30e4)

Two more things. Firstly, can you explain to me what is the benefit of exposing an Apple TV accessory as a set-top box accessory?

Secondly, in regard to the pyatv issue I can confirm that an active bluetooth connection also reports the AppleTV as powered on even if it is actually paused/off.

Best
-nicozagi

**Comment by maxileith** [comment]
> I was wondering if there is a way to exclude the AppleTVs from Homebridge from the iPhone remote dropdown menu? Having each device twice can be confusing. Perhaps I also misunderstand the whole concept and I am supposed to kick the original AppleTVs from the Home app?

This is not possible. Television devices always expose a remote. Removing the original Apple TV from HomeKit won't make the original Apple TV disappear in the remote app, since the app displays all Apple TVs in the network (doesn't matter if they are connected with HomeKir or not).

> Two more things. Firstly, can you explain to me what is the benefit of exposing an Apple TV accessory as a set-top box accessory?

See #191.

**Reply by nicozagi** [reply]
🙏 Thanks mate!


Cheers!
